packages/appsync-emulator-serverless/schema.js

```diff
@@ -187,31 +188,58 @@ const dispatchRequestToSource = async (
   }
 };
 
+const batchLoaders = {};
```

Comment:
I am not a fan of having these be global. Is there another place we can put these (such as a new object in generateTypeResolve along with requestPath?)

Reply:
Yeah, I guess we could move that somewhere in a better place.
Ideally, the `batchRequests` should be injected into the `load` function (instead of a `key`), but the dataloader does not work like that. 
An option would be a custom implementation of the dataloader but I did not want to overcomplicate this either.

data loaders could probably go in `generateTypeResolve()`, that sounds like the right thing to do.